base/doc/ltnews39.tex

```diff
@@ -602,6 +605,17 @@ \subsection{\pkg{array}, \pkg{longtable}, \pkg{tabularx}: support tagging}
 \texttt{latex-lab} patching the packages when tagging was requested.
 
 
+\subsection{\pkg{array}: No \cs{unskip} in math cells}
+
+Math cells in the standard \env{array} environment of the kernel are
+not subject to space removal at the right end of the cell, i.e., explicit
+spaces from \cs{hspace} or \verb*|\ |, etc.\ are honored
+(normal spaces are automatically ignored in math). In the \pkg{array}
+package all spaces got removed by calling \cs{unskip} unconditionally,
+regardless of the type of cell. This difference in behavior has been
+now been corrected.
```

Comment:
duplicate "been"

"has been now been corrected" -> "has now been corrected"

Reply:
@FrankMittelbach  clearly shouldn't have pinged an Englishman to proof read English text, thanks:-)